### PR TITLE
DOCSP-44904 Shorten TOC Labels

### PR DESCRIPTION
**Pull request by lindseymoore**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kotlin-sync/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-44904>
Staging - <https://deploy-preview-55--docs-kotlin-sync.netlify.app/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/connect.txt [hunk 1]
```diff
@@ -22,11 +22,11 @@ Connect to MongoDB
...
-   /connect/tls
-   /connect/stable-api
+   Create a MongoClient </connect/mongoclient>
+   Choose a Connection Target </connect/connection-targets>
+   Connection Options </connect/connection-options>
+   Enable TLS </connect/tls>
+   Stable API </connect/stable-api>
 
 ..    /connect/network-compression
 ..    /connect/server-selection
```
Comment (ltran-mdb2) [inline source/connect.txt:32]: Can we add labels to these as well?

File: source/write-operations.txt [hunk 2]
```diff
@@ -22,12 +22,12 @@ Write Data to MongoDB
...
-   /write/transactions
+   Insert </write/insert>
+   Update </write/update>
+   Replace </write/replace>
+   Delete </write/delete>
+   Bulk Write </write/bulk-write>
+   Transactions </write/transactions>
 
 .. 
    /write/gridfs
```
Comment (ltran-mdb2) [inline source/write-operations.txt:33]: Does this one need a label too?

File: source/index.txt [hunk 3]
```diff
@@ -14,19 +14,19 @@
...
-   /compatibility
-   Validate Driver Artifact Signatures </validate-signatures>
-   /whats-new
-   /issues-and-help
+   Connect </connect>
+   Write Data </write-operations>
+   Read Data </read>
+   Indexes </indexes>
+   Data Aggregation </aggregation>
+   Aggregation Expression Operations </agg-exp-ops>
```
Comment (ltran-mdb2) [inline source/index.txt:22]:
```suggestion
   Aggregation Operations </agg-exp-ops>
```

**Review by ltran-mdb2 — COMMENTED** [review]
Thanks for this work @lindseymoore ! I left a couple comments and questions for your consideration.

File: source/connect.txt (hunk 1 above)
Comment (lindseymoore) [reply source/connect.txt:32]: These are pages that haven't been created yet, so not necessary! @ltran-mdb2

File: source/write-operations.txt (hunk 2 above)
Comment (lindseymoore) [reply source/write-operations.txt:33]: Same as above

**Review by ltran-mdb2 — APPROVED** [review]
LGTM!

**Pull request merged**